src/makeRobot.js

```diff
@@ -38,7 +38,75 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const evacuateX = 1400;
+  const evacuateY = 500;
+  const robot = {
+    name,
+    wheels,
+    version,
+    x: 0,
+    y: 0,
```

Comment:
x, y краще зберігати в об'єкту coords
```suggestion
    coords: {
      x: 0,
      y: 0,
    }
```